website/documentation/getting-started/observability/components.md

```diff
@@ -36,13 +40,24 @@ The password you can use to log in can be retrieved as shown in the image below.
 
 ![](./images/access-plutono.png)
 
-After logging into Plutono, you will see dashboards like the one below. The depicted dashboard, for example, offers an overview of the Kubernetes Control Plane availability.
+### Accessing the dashboards
+
+After logging in you will be greeted with a Plutono welcome screen. Click on `General/Home` at the top as depicted with the red arrow in the next picture:
 
+![](./images/welcome_plutono.png)
```

Comment:
I can't seem to find this image in the PR, or any of the other newly added images.